src/models.ts

```diff
@@ -449,7 +449,7 @@ export const SORT_OPTIONS: Record<SortField, SortOption> = {
     handledBySearchService: false,
     searchServiceKey: 'favoritedate',
     displayName: 'Date favorited',
-    urlNames: ['favoritedate'],
+    urlNames: ['favoritedate', 'date_favorited'],
```

Comment:
The PPS identifies this sort using `date_favorited`, a format that differs from the other date sorts because of the different backing store for favorites & date favorited. Adding the new one so that we recognize it correctly, and leaving the old one for backwards-compatibility.